cython/docs/how-to-run.md

```diff
@@ -0,0 +1,46 @@
+# How to Run
+
+## Intro
+The compute.pyx module calculates the chi squared value using an input of 3 numpy arrays (x, y, and sigma). Since it uses cython, it must first be compiled.
+
+First make sure you have cython and setuptools installed:
+```bash
+pip install --upgrade setuptools cython
```

Comment:
can you add a requirements file that includes the two packages

Reply:
Sure thing. Pushed in PR #22